blog/2024-12-10-pydantic.md

```diff
@@ -0,0 +1,14 @@
+---
+author: Joost Lekkerkerker
+authorURL: https://github.com/joostlek
+authorImageURL: https://avatars.githubusercontent.com/u/7083755?v=4
+title: "Pydantic bump to v2"
+---
+
+Pydantic is a widely used library in Python for data validation. On June 30, 2023, Pydantic v2 was released, introducing significant changes that are not backward compatible with Pydantic v1.
+
+Home Assistant Core 2024.12 continues to ship with the Pydantic v1 library. Over the past year, our community has worked hard to ensure that the libraries used by Home Assistant Core are compatible with both Pydantic v1 and v2. This dual compatibility has helped make our transition to Pydantic v2 as smooth as possible.
```

Comment:
```suggestion
Over the past year, our community has worked hard to ensure that the libraries used by Home Assistant Core are compatible with both Pydantic v1 and v2. This dual compatibility has helped make our transition to Pydantic v2 as smooth as possible.
```
This reads a bit off, perhaps it's not necessary at ll?

Reply:
We still have a few subreleases until the end of the year, so I think it might help being explicit here that this only applies from 2025.1 onwards